R/git.R

```diff
@@ -201,12 +179,9 @@ git_delete_sample_log <- function(folder_path="/tmp"){
                    stderr = FALSE)
 }
 
-
-
-#' Git Init
+#' Initialize a git repo in a folder.
```

Comment:
This is reverting the changes on master. Can you leave the code on master as is?